_author_instructions/04_software_analyses.md

```diff
@@ -1,28 +1,29 @@
 ---
 layout: single
 sidebar:
-  nav: authors_compare_simulations.md
-title: Comparisons of Molecular Simulation Packages
-excerpt: What is a publishable comparison for molecular simulation packages?
-permalink: /authors/compare_simulations/
+  nav: authors_software_analyses.md
+title: Software Analyses
+excerpt: What is a publishable software analyis for molecular simulation packages?
+permalink: /authors/software_analyses/
 ---
 
-## What is a publishable comparison for molecular simulation packages?
+## What is a publishable software analysis paper for molecular simulation packages?
 
-Simulation comparison papers describe attempts to perform the same
-calculation with a range of different simulation programs. 
+Software analysis papers describe attempts to perform the same
+calculation with a range of different simulation programs, perform anaysis of changes in perfomrmance over a series of releases of a single software package.  In some cases, comparisons of the same calculation with different methods may also be appropriate for a software analysis paper.
```

Comment:
"OR perform analysis of changes"
Do we want to state that such analyses are intended for packages with a significant publication record and user base?  Or is it ok to compare a new MD package to gromacs, for example?

Reply:
Added some suggestions.